server/controllers/admin/employees.js

```diff
@@ -0,0 +1,313 @@
+    'creditor.uuid as creditor_uuid, creditor.text AS creditor_text, creditor.group_uuid as creditor_group_uuid, creditor_group.account_id ' +
+    'FROM employee ' +
+    ' JOIN grade ON employee.grade_id = grade.uuid ' +
+    ' JOIN fonction ON employee.fonction_id = fonction.id ' +
+    ' JOIN debitor ON employee.debitor_uuid = debitor.uuid ' +
+    ' JOIN creditor ON employee.creditor_uuid = creditor.uuid ' +
+    ' JOIN creditor_group ON creditor_group.uuid = creditor.group_uuid ' +
+    ' ORDER BY employee.name ASC, employee.postnom ASC, employee.prenom ASC';
+
+  db.exec(sql)
+  .then(function (rows) {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+};
+
+/**
+* Get list of availaible holidays for an employee
+*/
+exports.listHolidays = function (req, res, next) {
+  var pp = JSON.parse(req.params.pp);
+  var sql =
+    'SELECT holiday.id, holiday.label, holiday.dateFrom, holiday.percentage, holiday.dateTo ' +
+    'FROM holiday WHERE ' +
+    '((holiday.dateFrom >= ? AND holiday.dateFrom <= ?) OR ' +
+    '(holiday.dateTo >= ? AND holiday.dateTo <= ?) OR ' +
+    '(holiday.dateFrom <= ? AND holiday.dateTo >a ?)) AND ' +
+    'holiday.employee_id = ? ;';
+
+  var data = [
+    pp.dateFrom, pp.dateTo,
+    pp.dateFrom, pp.dateTo,
+    pp.dateFrom, pp.dateFrom,
+    req.params.employee_id
+  ];
+
+  db.exec(sql, data)
+  .then(function (rows) {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+};
+
+/**
+* Check an existing holiday
+*/
+exports.checkHoliday = function (req, res, next) {
+  var sql =
+    'SELECT id, employee_id, label, dateTo, percentage, dateFrom FROM holiday WHERE employee_id = ?' +
+    ' AND ((dateFrom >= ?) OR (dateTo >= ?) OR (dateFrom >= ?) OR (dateTo >= ?))' +
+    ' AND ((dateFrom <= ?) OR (dateTo <= ?) OR (dateFrom <= ?) OR (dateTo <= ?))';
+
+  var data = [
+    req.query.employee_id,
+    req.query.dateFrom, req.query.dateFrom, req.query.dateTo, req.query.dateTo,
+    req.query.dateFrom, req.query.dateFrom, req.query.dateTo, req.query.dateTo
+  ];
+
+  if (req.query.line !== '') {
+    sql += ' AND id <> ?';
+    data.push(req.query.line);
+  }
+
+  db.exec(sql, data)
+  .then(function (rows) {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+};
+
+/**
+* Check an existing offday
+*/
+exports.checkOffday = function checkHoliday(req, res, next) {
+  var sql ='SELECT * FROM offday WHERE date = ? AND id <> ?';
+  db.exec(sql, [req.query.date, req.query.id])
+  .then(function (rows) {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+};
+
+/**
+* Returns an object of details of an employee referenced by an `id` in the database
+*
+* @param {object} request The express request object
+* @param {object} response The express response object
+* @param {object} next The express middleware next object
+*
+* @example
+* // GET /employees/:id : Get details of an employee
+* var employees = require('categorised/employees');
```

Comment:
This comment should read `require('admin/employees')` not `require('categorised/employees')`.